### DISCUSSION TITLE
Planning settimanale, compiti/arg. di lezione

### DISCUSSION THREAD

**Discussion by gixxvanni** [iisgiua/giuaschool#618] [Idee e proposte]
Si potrebbe inserire la funzione Planning Settimanale e la possibilità di inserire i compiti direttamente dall'interfaccia degli argomenti di lezione.

**Comment by trinko** [comment]
La segno come richiesta.

**Comment by ghost** [comment]
@gixxvanni Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
